linode/manager
# refactor: [M3-6325] – MUI v5 Migration - `SRC > Components > DocumentTitle`

## Description 📝
Bring `DocumentTitle.tsx` up to speed with our current code conventions and restore the dynamic document title functionality that was accidentally lost after a [recent refactor](https://github.com/linode/manager/pull/9256).

## Major Changes 🔄
- Refactor class components and old patterns out of `DocumentTitle.tsx`
- Restore dynamic document title functionality 🎉 

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-07-14 at 10 56 18 AM](https://github.com/linode/manager/assets/114682940/8e11d774-2fe6-437c-b77e-1fa375ead484) | ![Screenshot 2023-07-14 at 10 56 54 AM](https://github.com/linode/manager/assets/114682940/0366e471-97f1-4775-ae8b-a17ff9000299) |

## How to test 🧪
Go through the app and ensure that the tab name gets updated with the appropriate section. For example, on the Linodes landing page, you should see `Linodes | Akamai Cloud Manager`.
